Csv/CsvReader.cs

```diff
@@ -180,12 +184,19 @@ private static void InitializeOptions(string line, CsvOptions options)
             lock (syncRoot)
             {
                 if (!splitterCache.TryGetValue(options.Separator, out splitter))
-                    splitterCache[options.Separator] = splitter = new Regex(string.Format(@"(?>(?(IQ)(?(ESC).(?<-ESC>)|\\(?<ESC>))|(?!))|(?(IQ)\k<QUOTE>(?<-IQ>)|(?<QUOTE>"")(?<IQ>))|(?(IQ).|[^{0}]))+|^(?={0})|(?<={0})(?={0})|(?<={0})$", Regex.Escape(options.Separator.ToString())), (RegexOptions)8);
+                    splitterCache[options.Separator] = splitter = CreateRegex(options);
             }
 
             options.Splitter = splitter;
         }
 
+        private static Regex CreateRegex(CsvOptions options)
+        {
+            // Since netstandard1.0 doesn't include RegexOptions.Compiled, we include it by value (in case the target platform supports it)
```

Comment:
I made a guess as to why the number 8 was explicitly included after trying to use `RegexOptions.Compiled` and getting an error message.  I figured adding a comment about why this was done would be helpful, and so I made it its own method.